docs/curriculum-topics-resources/mentorship-resources.md

```diff
@@ -4,132 +4,132 @@ sidebar_position: 1
 
 # Mentorship & Resources for Startups
 
-## Use UNICEF’s Open Source Inventory 
+## Use UNICEF’s Open Source Inventory
 
-Leverage this [Inventory](https://unicef.github.io/inventory/meta/mentorship/onboarding/) as a self serve and directory of resources and “How-to’s” for a startup looking to and/or operating on an open source business. 
+Leverage this [Inventory](https://unicef.github.io/inventory/meta/mentorship/onboarding/) as a self serve and directory of resources and “How-to’s” for a startup interested in or operating on an open source business.
 
 The Open Source Inventory is a knowledge-base of best practices around creating and working with Open Source works. This is an important resource for the Open Source Mentorship and was created based on experiences of previous graduates of the UNICEF Innovation Fund. You can also request new topics to be added to the knowledge-base too.
 
-## Topics covered in Open Source Mentorship
-These modules are offered as part of the Innovation Fund Open Source Mentorship.  More can be found in the [UNICEF O.S. Inventory Mentorship Modules](https://unicef.github.io/inventory/meta/mentorship/modules/). 
+## Topics Covered in Open Source Mentorship
+These modules are offered as part of the Innovation Fund Open Source Mentorship.  More can be found in the [UNICEF O.S. Inventory Mentorship Modules](https://unicef.github.io/inventory/meta/mentorship/modules/).
 
-### Legal & Licensing: Foundations of Open Source Intellectual Property 
```

Comment:
I noticed "Intellectual Property" or IP is mentioned a lot in the guide. I left most of it in the section on licenses, but I removed it here since open source law is built on copyright law and not IP law. However, I am not a lawyer...